examples/00-EDB/11_post_layout_parameterization.py

```diff
@@ -5,40 +5,37 @@
 """
 
 ###############################################################################
-# Define input parameters
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Define input parameters.
 signal_net_name = "DDR4_ALERT3"
 coplanar_plane_net_name = "1V0"  # Specify coplanar plane net name for adding clearance
 layers = ["16_Bottom"]  # Specify layers to be parameterized
 
 ###############################################################################
-# Perform required imports
-# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports.
 import os
 import tempfile
 import pyaedt
 
-from pyaedt import downloads
-from pyaedt import Edb
-
-temppath =  pyaedt.generate_unique_folder_name()
+temp_dir = tempfile.TemporaryDirectory(suffix=".ansys")
 
 ###############################################################################
-# Download and open example layout file in edb format
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-edb_fpath = pyaedt.downloads.download_file('edb/ANSYS-HSD_V1.aedb',destination=temppath)
-appedb = Edb(edb_fpath, edbversion="2023.2")
+# Download and open example layout file in edb format.
+edb_path = pyaedt.downloads.download_file('edb/ANSYS-HSD_V1.aedb',
+                                           destination=temp_dir.name)
+edb = pyaedt.Edb(edb_path, edbversion="2023.2")
 
 ###############################################################################
-# Cutout
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-appedb.cutout([signal_net_name], [coplanar_plane_net_name, "GND"],
+# ## Cutout
+#
+# The ``Edb.cutout()`` method takes a list of
+# signal nets as the first argument and a list of
+# reference nets as the 2nd argument.
+edb.cutout([signal_net_name], [coplanar_plane_net_name, "GND"],
               remove_single_pin_components=True)
 
 ###############################################################################
-# Get all trace segments from the signal net
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-net = appedb.nets[signal_net_name]
+# Retrive the path segments from the signal net.
```

Comment:
```suggestion
# Retrieve the path segments from the signal net.
```
We've really got to standardize on either the inclusion or exclusion of periods!